target/linux/ramips/image/Makefile

```diff
@@ -79,6 +79,17 @@ define Build/loader-kernel
 	$(call Build/loader-common,LOADER_DATA="$@")
 endef
 
+define Build/loader-okli-compile
+	$(call Build/loader-common,FLASH_OFFS=$(LOADER_FLASH_OFFS))
```

Comment:
`DEVICE_VARS` should be extended with `LOADER_FLASH_OFFS` to avoid fixing commit like 111029bea7b5c1075f7aa12e8431fe840c3dca12!

Reply:
Yes, agreed! Do you want me to add it here (not really used yet in general), or include it in the (imminent 😄) PR coming for the RAVPower WD-03 (where it is already added)? Thanks for catching this!

Reply:
It needs to be included to the commit where the new `Build/loader-okli-compile` recipe appears.

Reply:
That makes sense, no issue here. I'll add in the two variables, and try a new push 😆.

So LOADER_FLASH_OFFS and LOADER_TYPE, agreed? Actually, LOADER_TYPE is already there, will add LOADER_FLASH_OFFS. Also add in LZMA_TEXT_START at the same time, or hold that one for now?

Reply:
No need for touching `LZMA_TEXT_START` for a 64 MB RAM device! :wink:

Reply:
Good point 🤣. Added LOADER_FLASH_OFFS, force pushed. Hopefully not git-mess again ... LOL.

Thanks!